### PR TITLE
Implement CV monochrome force

### PR DESCRIPTION
**Pull request by juanpms2**
Ready to review

### CODE REVIEW

File: apps/react-app/src/pods/template-export/components/export-config/export-config.component.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ interface Props {
 }
 
 const DOWNLOAD_MESSAGE_TIMEOUT = 2500;
-const DESING_OPTIONS: TemplateCV[] = ['Sith Elegance', 'Galactic CleanTech'];
+const DESING_OPTIONS: TemplateCV[] = ['Sith Elegance', 'Galactic CleanTech', 'Monochrome Force'];
```
Comment (brauliodiez) [inline apps/react-app/src/pods/template-export/components/export-config/export-config.component.tsx:20]: Typo: DESIGN_OPTIONS

File: export-samples/html/cv-monochrome-force.html [hunk 2]
```diff
@@ -1032,13 +1037,18 @@ <h2 class="title">Educación</h2>
 
         if (window.innerWidth > 834) {
           // Mueve los elementos a las columnas correspondientes
```
Comment (brauliodiez) [inline export-samples/html/cv-monochrome-force.html:1039]: Not sure if we are commenting in english or spanish, if it's english move it to english

File: packages/manfred-common/src/doc-parts/profile-section/profile-section.mapper.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { ManfredAwesomicCV } from '@/model';
-import { ProfileSectionVm, RelevantLinkVm, ManfredRelevantLink } from './profile-section.vm';
+import { ProfileSectionVm, RelevantLinkVm, ManfredRelevantLink, PhoneNumbers } from './profile-section.vm';
 
+//Sólo afecta al .json de David Bonilla
```
Comment (brauliodiez) [inline packages/manfred-common/src/doc-parts/profile-section/profile-section.mapper.ts:4]: Please in english and ellaborate about the issue

File: apps/react-app/src/pods/template-export/components/export-config/export-config.component.tsx (hunk 1 above)
Comment (juanpms2) [reply apps/react-app/src/pods/template-export/components/export-config/export-config.component.tsx]: it's fixed

File: export-samples/html/cv-monochrome-force.html (hunk 2 above)
Comment (juanpms2) [reply export-samples/html/cv-monochrome-force.html]: removed comments, not neccesary

File: packages/manfred-common/src/doc-parts/profile-section/profile-section.mapper.spec.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import { ManfredAwesomicCV } from '@/model';
+import { ContactMean, ManfredAwesomicCV } from '@/model';
```
[the remark this answers is not in the document]
Comment (juanpms2) [reply packages/manfred-common/src/doc-parts/profile-section/profile-section.mapper.spec.ts:1]: It's from the manfred's model, not ours

File: packages/manfred-common/src/doc-parts/profile-section/profile-section.mapper.ts (hunk 3 above)
Comment (juanpms2) [reply packages/manfred-common/src/doc-parts/profile-section/profile-section.mapper.ts]: it's fixed

**Pull request merged**